packages/amplify/amplify_flutter/example/android/app/build.gradle

```diff
@@ -40,7 +40,7 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.amazonaws.amplify.amplify_flutter_example"
-        minSdkVersion 21
+        minSdkVersion 24
```

Comment:
Are we going to merging all of this migration code into a separate dev-preview branch, or migrating everything into main and then keeping the dev-preview on a seperate branch where the new packages are enabled view pubspecs?